static/js/lib/validation/profile.js

```diff
@@ -299,5 +299,10 @@ export const validateFinancialAid = (edit: FinancialAidState): FinancialAidValid
   if (!edit.checkBox) {
     errors['checkBox'] = 'You must agree to these terms';
   }
+
+  if (edit.income && !/^\d+$/.test(edit.income)) {
```

Comment:
I'm not sure this is exactly the regex we want - I think it might be more clear to test for non-digit characters, instead of `!`-ing a test for all digits.

so maybe something like `/\D+/` ?